### PR TITLE
Add YAML file type icon

### PR DESCRIPTION
**Pull request by DevThurain**
Add YAML file type icon from [file-icons/icons](https://github.com/file-icons/icons)

https://github.com/file-icons/icons/blob/master/svg/YAML.svg
Release Notes:
- Added YAML file type icon.

### CODE REVIEW

**Comment by DevThurain** [comment]
@cla-bot check

File: assets/icons/file_icons/file_types.json [hunk 1]
```diff
@@ -109,7 +109,7 @@
         "xls": "document",
         "xlsx": "document",
         "xml": "template",
-        "yaml": "settings",
+        "yaml": "yaml",
         "yml": "settings",
```
Comment (d1y) [inline assets/icons/file_icons/file_types.json:113]:
```suggestion
"yml": "yaml"
```

**Comment by DevThurain** [comment]
@cla-bot check

**Comment by DevThurain** [comment]
@cla-bot check

**Review by d1y — APPROVED** [review]

**Comment by SomeoneToIgnore** [comment]
Thank you for the PR and the reviews!

**Pull request merged**